examples/step-29/doc/intro.dox

```diff
@@ -96,26 +96,26 @@ boundary condition will yield partial reflections, i.e. only parts of the wave
 will pass through the boundary as if it wasn't here whereas the remaining
 fraction of the wave will be reflected back into the domain.
 
-If we are willing to accept this as a sufficient approximation to an absorbing boundary we finally arrive at the following problem for $u$: 
+If we are willing to accept this as a sufficient approximation to an absorbing boundary we finally arrive at the following problem for $u$:
```

Comment:
Want to break the line around column 80?

Reply:
I wrapped this and all the other long lines (and long equations) .in this file.